app/views/application/_navbar.html.haml

```diff
@@ -11,18 +11,38 @@
       %ul.nav.navbar-nav
         %li
           = link_to 'Home', root_path
-          - if current_user.try(:customer?)
-            %li
-              = link_to 'Edit profile', edit_user_registration_path
-          - if current_user.try(:owner?)
-            %li
-              = link_to 'Create restaurant', new_restaurant_path
-            %li
-              = link_to 'Create menu', new_menu_path
-            %li
-              = link_to 'Create dish', new_dish_path
-        %li
-          - unless current_user.try(:owner?)
+        - if current_user
+          %li
+            = link_to 'Your profile', edit_user_registration_path
+        - if current_user.try(:owner?)
```

Comment:
Again, we are using abilities for this kind of conditions. Please go over the documentation for CanCanCan.